### PR TITLE
First steps to improve the documentation for Windows

### PR DESCRIPTION
**Pull request by kalbermattenm**
This PR rewrites the documentation for Windows.

There are still things to add like downloading wheels, which pip cannot install because of binaries and dll, but this will be part of another PR (which I hope to propose soon...)

Please review

### CODE REVIEW

File: doc/integrator/requirements.rst [hunk 1]
```diff
@@ -58,22 +58,51 @@ to change it set the ``PRINT_OUTPUT`` value in the makefile.
...
-<http://www.stickpeople.com/projects/python/win-psycopg/>`_.
-
-When you download and configure Apache be sure that modules ``header_module``,
-``expire_module`` and ``rewrite_module`` are uncommented. You must also download
-and add modules ``mod_wsgi`` (http://modwsgi.readthedocs.org/) and ``mod_fcgid``
-(https://httpd.apache.org/mod_fcgid/).
+GeoMapFish can be used under Windows, but you will need to install Cygwin
+and some additional stuff for Python.
+
+We assume that the used Python version is 2.7
```
Comment (asaunier) [inline doc/integrator/requirements.rst]: We assume that Python 2.7 is used.

File: doc/integrator/requirements.rst [hunk 2]
```diff
@@ -58,22 +58,51 @@ to change it set the ``PRINT_OUTPUT`` value in the makefile.
...
+
+Once Python installed, you should add two folders to your ``path`` environment
```
Comment (asaunier) [inline doc/integrator/requirements.rst]: Once Python is installed

File: doc/integrator/requirements.rst [hunk 3]
```diff
@@ -58,22 +58,51 @@ to change it set the ``PRINT_OUTPUT`` value in the makefile.
...
+* Run the downloaded installer
+* Choose an install folder not containing any space or weird characters (like parentheses)
+
+When installing Cygwin, please make sure to install the following non-default packages:
+
+* ``make`` from the ``devel`` folder
+* ``gettext-devel`` from the ``devel`` folder
+* ``wget`` from the ``web`` folder
+
+Cygwin should *always* in administrator mode. To configure that:
```
Comment (asaunier) [inline doc/integrator/requirements.rst]: Missing verb after "should always"

File: doc/integrator/requirements.rst [hunk 4]
```diff
@@ -58,22 +58,51 @@ to change it set the ``PRINT_OUTPUT`` value in the makefile.
...
+
+* Go to the Windows start menu
+* Right-clic the Cygwin Terminal program icon
```
Comment (asaunier) [inline doc/integrator/requirements.rst]: clic => click

File: doc/integrator/requirements.rst [hunk 5]
```diff
@@ -58,22 +58,51 @@ to change it set the ``PRINT_OUTPUT`` value in the makefile.
...
+* Open its properties
+* Under the ``Compatibility`` tab, check the ``Run this program as an administrator``
+
+Configure Git
+^^^^^^^^^^^^^
+
+Cygwin comes with its own git distribution. It might be a really good idea to configure
```
Comment (asaunier) [inline doc/integrator/requirements.rst]: maybe "package" instead of "distribution"?

**Comment by kalbermattenm** [comment]
Thanks @asaunier, I applied your review.

**Comment by asaunier** [comment]
Thanks for the changes. Looks good to me.
@sbrunner please merge if it is OK for you.

**Comment by sbrunner** [comment]
Thanks @kalbermattenm, @asaunier

**Pull request merged**